merlin/features/array/cudf.py

```
@@ -0,0 +1,92 @@
+        """
+
+        @classmethod
+        def array_type(cls):
+            return cudf.Series
+
+        @classmethod
+        def convert_to_array(cls, other):
+            return other.to_numpy()
+
+        @classmethod
+        def convert_to_cuda_array(cls, other):
+            raise NotImplementedError
+
+        @classmethod
+        def convert_to_dlpack_capsule(cls, other):
+            return other.to_dlpack()
+
+        def build_from_cuda_array(self, other):
+            """
+            Build a cudf.Series from an object that implements the Cuda Array Interface.
+
+            Parameters
+            ----------
+            other : array-like
+                The array-like object to build the Series from
+
+            Returns
+            -------
+            cudf.Series
+                The Series built from the array-like object
+            """
+            return cudf.Series(other)
+
+        def build_from_array(self, other):
+            """
+            Build a cudf.Series from an object that implements the Numpy Array Interface.
+
+            Parameters
+            ----------
+            other : array-like
+                The array-like object to build the Series from
+
+            Returns
+            -------
+            cudf.Series
+                The Series built from the array-like object
+            """
+            return cudf.Series(other)
+
+        def build_from_dlpack_capsule(self, capsule):
+            """
+            Build a cudf.Series from an object that implements the DLPack Standard.
+
+            Parameters
+            ----------
+            other : array-like
+                The array-like object to build the Series from
+
+            Returns
+            -------
+            cudf.Series
+                The Series built from the array-like object
+            """
+            return cudf.io.from_dlpack(capsule)
+
+
+MerlinCudfArray = None if not cudf else _MerlinCudfArray
```

Could we have a class that raises on init instead of None, so that users get an error telling them that the corresponding package is not installed?